Source/Extensions/Blazorise.DataGrid/DataGrid.razor.cs

```diff
@@ -668,6 +704,8 @@ protected IEnumerable<TItem> DisplayData
         /// </summary>
         [Parameter] public RenderFragment NextPageButtonTemplate { get; set; }
 
+        [Parameter] public bool IsLoading { get; set; }

```

Comment:
Maybe `IsLoading` doesn't need to be Parameter?

Reply:
Yeah, i think, i remove the annotation and make it protected.